Homework 5-6

```diff
@@ -0,0 +1,98 @@
+HOMEWORK WEEK 5-6 
+(handout for students) 
+TASK 1 (Agile Techniques) 
+Question 1
+Complete definitions for Scrum related key terminology provided below. 
+SCRUM CEREMONIES 
+·  Product backlog refinement is a type of meeting held toward the end of the sprint to check that the backlog has been achieved before the next one
+•	Sprint planning, this is when you prioritise the tasks in a sprint and plan the order they are going to be actioned
+•	  Daily scrum , this is held at the start of the day to see the progress in a sprint held by a scrum master.
+•	  Sprint review, this is where you review the work produced by the sprint and whether further additions need to be made on backlog tasks
+•	  Sprint retrospective, this is after the sprint where you discuss what was useful/ not useful to apply to the next sprints in the future. 
+SCRUM ROLES 
+•	·  ScrumMaster , is a person who follows the framework of the scrum to ensure tasks get completed
```

Comment:
It is very key to mention what they do with respect to the team and in the scrum framework. 

The scrum master is the facilitator for the scrum team who manages how information is exchanged, organises the scrum team, and arranges and facilitates the team's meetings. They are the prime communicator for the team ensuring that the team dynamic is well, establishing a good connection between the team and product owner/stakeholders and clearing obstacles. Has no authority over the team - more like a coach than a manager.